webdriver-extension/index.bs

```diff
@@ -0,0 +1,532 @@
+associated with the usage of the <a>AbsoluteOrientationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary AbsoluteOrientationReadingSet {
+        required FrozenArray&lt;double>? quaternion;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Relative Orientation Sensor ### {#mock-relative-orientation-sensor}
+The <dfn for="MockSensorType" enum-value>"relative-orientation"</dfn> type is the mock sensor type
+associated with the usage of the <a>RelativeOrientationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary RelativeOrientationReadingSet : AbsoluteOrientationReadingSet {
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Geolocation Sensor ### {#mock-geolocation-sensor}
+The <dfn for="MockSensorType" enum-value>"geolocation"</dfn> type is the mock sensor type
+associated with the usage of the <a>GeolocationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary GeolocationReadingSet {
+        required double? latitude;
+        required double? longitude;
+        required double? altitude;
+        required double? accuracy;
+        required double? altitudeAccuracy;
+        required double? heading;
+        required double? speed;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Proximity Sensor ### {#mock-proximity-sensor}
+The <dfn for="MockSensorType" enum-value>"proximity"</dfn> type is the mock sensor type
+associated with the usage of the <a>ProximitySensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Set</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary ProximityReadingSet {
+        required double? distance;
+        required double? max;
+        required boolean? near;
+      };
+    </pre>
+  </dd>
+</dl>
+
+Extensions {#webdriver-extensions}
+=====================
+
+## Commands ## {#mock-sensor-commands}
+
+### Create mock sensor ### {#create-mock-sensor-command}
+
+<table>
+  <tbody>
+    <tr>
+      <th>HTTP Method</th>
+      <th><a lt="extension command URI template">URI Template</a></th>
+    </tr>
+    <tr>
+      <td>POST</td>
+      <td>/session/{session id}/sensor</td>
+    </tr>
+  </tbody>
+</table>
+
+The <dfn>Create mock sensor</dfn> <a>extension command</a> simulates user
+creation of a <a>mock sensor</a>.
+
+The <a>remote end steps</a> are:
+1. Let |configuration| be the |configuration| argument, <a>converted to an IDL value</a>
+   of type {{MockSensorConfiguration}}. If this throws an exception, return a
+   <a>WebDriver error</a> with <a>WebDriver error code</a> <a>invalid argument</a>.
+2. Let |type| be the |configuration|.{{MockSensorConfiguration/mockSensorType}}, if <a>current browsing context</a>
+   already exists this |type| of [=mock sensor=], return a <a>WebDriver error</a> with <a>WebDriver error code</a>
```

Comment:
I think this should read "if the current browsing context already has this type" or similar?

But more importantly, this means that only one sensor of a given type can be created. I assume that's fine, but can you mention it somewhere more prose-y or in a note, so that people who try to use the API understand this quickly?